.travis.yml

```diff
@@ -36,14 +25,34 @@ before_install:
   - export PATH=$HOME/.yarn/bin:$PATH
 
 install:
-  - yarn install --no-lockfile --non-interactive
+  - yarn install --frozen-lockfile --non-interactive
+
+stages:
+  - test
+  - additional tests
+
+jobs:
+  fail_fast: true
+
+  include:
+    # runs tests with current locked deps and linting
+
+    - env: NAME=test # used only to make Travis UI show description
+      script:
+        - yarn test:node
+        - yarn test
 
-before_script:
-  - $BEFORE_SCRIPT
+    - env: NAME=test sauce
```

Comment:
Lets move this one into additional tests too.

Basically, I want a single job that runs in the first stage which is using the lockfile (e.g. no drift), then do the other two things in "additional tests"...